tt_metal/impl/device/device.cpp

```diff
@@ -3299,6 +3300,8 @@ bool Device::using_slow_dispatch() const {
 }
 
 void Device::begin_trace(const uint8_t cq_id, const uint32_t tid) {
```

Comment:
Do you think this must be renamed to "start_replay_capture" and "end_replay_capture"?